tests/test_worker_operation.py

```diff
@@ -0,0 +1,62 @@
+from src.worker_operation import WorkerOperation
```

Comment:
[STYLE] I'd create a separate test for each case instead of overloading the `phases` function